submissions/maryOkosun/databases/trans_company.sql

```diff
@@ -0,0 +1,39 @@
+-- created  a database called TRANS_COMPANY 
+CREATE DATABASE trans_company; 
+USE trans_company;
+
+-- created a table called PASSENGERS 
+CREATE TABLE passengers(
+    passenger_id int not null auto_increment,
+	full_name varchar(100) not null,
+    sex varchar(10) not null,
+    age int null,
```

Comment:
For `age` use `varchar` instead. And also, it is `null` by default. No need for specifying `null`

Reply:
Thanks @adaugochi 
I have corrected the data type of the 'age'.
Please review. Thanks